src/chainer.test.js

```diff
@@ -1 +1,31 @@
+'use strict';
+
 const { chainer } = require('./chainer');
+
+describe('chainer', () => {
+  it('should chain the unary functions correctly', () => {
+    const mockFunctionA = jest.fn((x) => x + 1);
```

Comment:
The jest.fn method is used for mocking a function. But in this context, it is unnecessary. Instead, you can just define these functions normally.